autoload/vundle/installer.vim

```diff
@@ -6,9 +6,16 @@
 " ...    -- any number of bundle specifications (separate arguments)
 " ---------------------------------------------------------------------------
 func! vundle#installer#new(bang, ...) abort
-  let bundles = (a:1 == '') ?
-        \ g:bundles :
-        \ map(copy(a:000), 'vundle#config#bundle(v:val, {})')
+  " No specific plugins are specified. Operate on all plugins.
+  if a:0 == 0
+    let bundles = g:bundles
+  " Specific plugins are specified for update. Update them.
+  elseif (a:bang)
+    let bundles = filter(copy(g:bundles), 'index(a:000, v:val.name) > -1')
```

Comment:
In this particular branch, if the `bundles` list turns out to be empty. Should we even start the installer? Maybe an error message could do? Right now it starts an empty installer.

Also, if some of the arguments are wrong (plugins I don't have), it will silently skip them.

I think we should at least treat the empty list case, maybe not the second case I described. Proper autocompletion should make that less likely.